build-configs/integreat-test-cms/assets/campaign-app-logo-mobile.png

```diff
@@ -0,0 +1 @@
+/Users/afischer/projects/integreat-app/build-configs/integreat/assets/campaign-app-logo-mobile.png
```

Comment:
please fix.

Reply:
i adjusted the symlink to use `integreat/assets` folder so if we add new assets the gonna be symlinked auto